mbuild/compound.py

```diff
@@ -136,6 +137,8 @@ def load(filename_or_object, relative_to_module=None, compound=None, coords_only
                 warn("More than one SMILES string in file, more than one SMILES "
                      "string is not supported, using {}".format(mymol.write("smi")))
 
+        # We create a temporary directory and mol2 file that will created from the smiles string
+        # A parmed structure and subsequenc mBuild compound will be created from this mol2 file
```

Comment:
```suggestion
        # A ParmEd structure and subsequent mBuild compound will be created from this mol2 file
```

Reply:
Once this change is applied, this can be merged.

Reply:
Just corrected